src/atoms/Photography/photography.module.css

```diff
@@ -0,0 +1,8 @@
+.photography {
```

Comment:
Please arrange properties in alphabetical order.

Reply:
Done.